extensions/shared/external-media/editor.scss

```diff
@@ -0,0 +1,363 @@
+/**
+ * External Media
+ */
+@import '../styles/gutenberg-base-styles.scss';
+
+$grid-size: 8px;
+
+@keyframes loading-fade {
```

Comment:
@retrofox  Keyframes end up in CSS global scope and should be unique. Recommend prefixing with `jetpack_`.

Reply:
Fixed in #16144.